### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by jorgesumle** [buckyroberts/Spider#43]
> This is an **open source**, multi-threaded website crawler written in Python. There is still a lot of work to do, so feel free to help out with development.

The license is missing. And so the code is proprietary. Here is the definition of open source: https://opensource.org/osd

@buckyroberts, can you please license it under a [free software license](https://www.gnu.org/licenses/license-list.html)?

**Comment by ghost** [comment]
https://github.com/AbdulSheikh/Spider

**Comment by jorgesumle** [comment]
What do you want to say with that link, @AbdulSheikh? Is that an answer?

**Comment by ghost** [comment]
The project has been moved to https://github.com/AbdulSheikh/Spider . This project is no longer supported In this github account.

**Comment by jorgesumle** [comment]
OK, but that doesn't solve the issue with licensing in any way.

I see that the project hasn't received a commit since last year and that the main contributor (@buckyroberts) is ignoring this issue completely, which renders this project unusable for many people without infringing copyright, with very little hope of this being fixed.

**Comment by ghost** [comment]
Yes, I moved the project over to the new github account because I can commit changes in minutes in not seconds.Please consider the new link a equal project with much/or more features.

**Comment by jorgesumle** [comment]
ping @buckyroberts
